src/_gettsim/transfers/kindergeld.py

```diff
@@ -174,3 +181,104 @@ def kind_bis_10_mit_kindergeld(
     """
     out = kindergeld_anspruch and (alter <= 10)
     return out
+
+
+def _kindergeld_per_child_m(
+    kindergeld_m: float,
+    kindergeld_anz_ansprüche: int,
+) -> float:
+    """Kindergeld per child.
+
+    Helper function for `kindergeld_zur_bedarfdeckung_m`. Returns the average Kindergeld
```

Comment:
Would revert the sentences: first what it calculates. Then (potentially) why/where we need it.